src/service/position-observer/position-observer-entry.js

```diff
@@ -107,8 +112,15 @@ export class PositionObserverEntry {
   /**
    * To update the position of entry element when it is ready.
    * Called when updateAllEntries, or when first observe an element.
+   * @param {boolean=} opt_force
    */
-  update() {
+  update(opt_force) {
+    if (!opt_force && this.turn != 0) {
+      this.turn--;
+      return;
+    }
+    this.turn = (this.fidelity == PositionObserverFidelity.LOW) ?
```

Comment:
This is slightly different than the old code. Now, when a force update comes in, all low fidelity Entries will be synchronized to the same frame turn. Using two nested if statements cleans it up:

```js
if (!opt_force) {
  if (this.turn > 0) {
    this.turn--;
    return;
  }
  
  // An if statement just seems nicer than the assignment conditional
  if (this.fidelity == LOW) {
    this.turn = 4;
  }
}

Reply:
Good point. Talked offline, will move back to old behavior. But we may need future discussion on whether we should randomize `entry.turn` or sync all low fidelity elements' measurement